app/assets/stylesheets/components/_published-dates.scss

```diff
@@ -15,3 +15,8 @@
   display: block;
   @include bold-16;
 }
+
+.app-c-published-dates__history {
+  padding-top: 10px;
```

Comment:
Should this be a variable?

Reply:
Maybe? There was a bare 10px for padding in the file, I just copied that. What would the variable be called if it were one?